pkg/operator/ceph/cluster/cluster.go

```diff
@@ -102,7 +102,7 @@ func (c *cluster) detectCephVersion(rookImage, cephImage string, timeout time.Du
 	}
 
 	job := versionReporter.Job()
-	job.Spec.Template.Spec.ServiceAccountName = "rook-ceph-cmd-reporter"
+	job.Spec.Template.Spec.ServiceAccountName = c.Namespace + "-cmd-reporter"
```

Comment:
wait, why would the service account name change when running in a different namespace?

Reply:
hum, maybe I went too far on my setup, I did a `sed s/rook-ceph/default/g` as I was testing multus and hit that error. I guess it changed the name of the SA too in the process. I can see that it makes sense to hardcode now... The SA is never supposed to change right?

Reply:
Agreed, I wouldn't expect the SA name to change.